_posts/2016-10-03-what-i-wish-i-knew.md

```diff
@@ -54,22 +54,22 @@ On the other hand, understand what are the tradeoffs of a given library. Is it s
 ##But still understand the fundamentals##
 Libraries are convenient, because they remove complexity. However, take the time to understand a bit of how they work under the hood.
 
-While your web framework will handle HTTP requests for you, you need to get what HTTP is and how to conveys information between two computers. Similarly, take the time to understand the basis of DNS, TLS, and other low-level protocols. If your application uses OAuth or session-based authentication, you need to know it and you need to know what the consequences are.
+While your web framework will handle HTTP requests for you, you need to get what [HTTP](https://en.wikipedia.org/wiki/Hypertext_Transfer_Protocol) is and how to conveys information between two computers. Similarly, take the time to understand the basis of [DNS](https://en.wikipedia.org/wiki/Domain_Name_System), [TLS](https://en.wikipedia.org/wiki/Transport_Layer_Security), and other low-level protocols. If your application uses [OAuth](https://en.wikipedia.org/wiki/OAuth) or [session-based authentication](https://www.owasp.org/index.php/Session_Management_Cheat_Sheet), you need to know it and you need to know what the consequences are. This page is a good starting point: [What happens when...](https://github.com/alex/what-happens-when).
```

Comment:
I would prefer
`[This page](https://github.com/alex/what-happens-when) is a good starting point.`
Or
`["What happens when..."](https://github.com/alex/what-happens-when) is a good starting point.`